k8s.gcr.io/images/k8s-staging-dns/promoter-manifest.yaml

```diff
@@ -0,0 +1,11 @@
+# google group for gcr.io/k8s-staging-dns is k8s-infra-staging-dns@kubernetes.io
+registries:
+- name: gcr.io/k8s-staging-dns
+  src: true
+- name: us.gcr.io/k8s-artifacts-prod/dns
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+- name: eu.gcr.io/k8s-artifacts-prod/dns
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+- name: asia.gcr.io/k8s-artifacts-prod/dns
+  service-account: k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com
+imagesPath: "../../images/k8s-staging-dns/images.yaml"
```

Comment:
This has to be split up into 2 files, the `promoter-manifest.yaml` and the `images.yaml`.

I will have to update the docs to reflect this.

Reply:
Actually, I see that the instructions https://github.com/kubernetes/k8s.io/tree/master/k8s.gcr.io#creating-staging-repos already state this.

What docs did you follow? I need to know so I can update them as well.

Reply:
I used the docs you linked. I did see that images.yaml should contain the images info and promoteer-manifest.yaml should contain the credentials. I did create images.yaml and left it empty since i do not need to promote an image right away. I was planning to send a separate PR when needed.

I got the content of the promoter-manifest.yaml file by looking at other staging repos examples, as the docs stated. Sorry if I missed something here. Which part of these contents should i have moved into images.yaml?

Reply:
For the promoter-manifest.yaml, if you look at e.g. https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/manifests/DO-NOT-MODIFY-legacy-backfill/promoter-manifest.yaml you can see how `imagesPath` does not exist.

Do you have a link to where you found the promoter-manifest.yaml that has the `imagesPath` still in it? I need to update that so that other people don't make the same mistake.

For the `images.yaml` you can use https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-apisnoop/images.yaml which has an empty file. You still need an empty file.

Reply:
Also, just in case you didn't notice it: you created `k8s.gcr.io/images/k8s-staging-dns/promoter-manifest.yaml` which is in the wrong path. It should be `k8s.gcr.io/manifests/k8s-staging-dns/promoter-manifest.yaml`.

Reply:
> For the promoter-manifest.yaml, if you look at e.g. https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/manifests/DO-NOT-MODIFY-legacy-backfill/promoter-manifest.yaml you can see how `imagesPath` does not exist.
> 
> Do you have a link to where you found the promoter-manifest.yaml that has the `imagesPath` still in it? I need to update that so that other people don't make the same mistake.
> 

I followed the example from k8s-staging-external-dns. It does not have the imagesPath now though. My mistake, i took the example from the first PR that added the repo - https://github.com/kubernetes/k8s.io/pull/479 , didn't realise the file was modified to remove 
> For the `images.yaml` you can use https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-apisnoop/images.yaml which has an empty file. You still need an empty file.

I have created an empty file. Will add some text indicating this is an empty file, so the diff shows up. Thanks!

Reply:
Awesome, thanks for the clarifications.